app/views/course/forum/topics/_topic.html.slim

```diff
@@ -0,0 +1,27 @@
+= content_tag_for(:tr, topic) do
+  td
+  td
+    h4
+      = link_to(topic.title, course_forum_topic_path(@course, @forum, topic))
+    div.started-by
+      = t('.started_by')
+      - topic_creator = current_course.course_users.where(user: topic.creator)
+      - if topic_creator
+        = link_to_user(topic_creator)
+      - else
+        = link_to_user(topic.creator)
+  td = topic.votes_count
+  td = topic.post_count
+  td = topic.view_count
+  td.latest-post
+    - last_post = topic.posts.first
+    - if last_post
+      = t('.last_post')
+      = link_to last_post.title, course_forum_topic_path(current_course, @forum, topic)
+      br
+      = t('.creator')
+      - last_post_creator = current_course.course_users.where(user: last_post.creator)
```

Comment:
@lowjoel Shall we override the `link_to_user` helper in course as described in #549 ?